clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/CramMetadata.scala

```diff
@@ -87,7 +86,39 @@ case class CramMetadata(
       // want to fixup files with just '.crai' when possible.
       craiPath = movedCram.map(
         cramUri => URI.create(s"$cramUri${CramExtensions.CraiExtensionAddition}")
-      )
+      ),
+      analysisFilesTxtPath =
+        pathMapper(analysisFilesTxtPath, CramExtensions.AnalysisFilesTxtExtension),
+      preAdapterSummaryMetricsPath = pathMapper(
+        preAdapterSummaryMetricsPath,
+        CramExtensions.PreAdapterSummaryMetricsExtension
+      ),
+      preAdapterDetailMetricsPath = pathMapper(
+        preAdapterDetailMetricsPath,
+        CramExtensions.PreAdapterDetailMetricsExtension
+      ),
+      alignmentSummaryMetricsPath = pathMapper(
+        alignmentSummaryMetricsPath,
+        CramExtensions.AlignmentSummaryMetricsExtension
+      ),
+      // Note preBqsrDepth Metrics no longer created by workflow
+      preBqsrSelfSmPath =
+        pathMapper(preBqsrSelfSmPath, CramExtensions.PreBqsrSelfSMExtension),
+      cramValidationReportPath = pathMapper(
+        cramValidationReportPath,
+        CramExtensions.CramValidationReportExtension
+      ),
+      crosscheckPath = pathMapper(crosscheckPath, CramExtensions.CrossCheckExtension),
+      fingerprintingSummaryMetricsPath = pathMapper(
+        fingerprintingSummaryMetricsPath,
+        CramExtensions.FingerprintingSummaryMetricsExtension
+      ),
+      fingerprintingDetailMetricsPath = pathMapper(
+        fingerprintingDetailMetricsPath,
+        CramExtensions.FingerprintingDetailMetricsExtension
+      ),
+      fingerprintVcfPath =
+        pathMapper(fingerprintVcfPath, CramExtensions.FingerprintVcfExtension)
```

Comment:
Reminder: there are more metrics files to fill in here before merging. I think everything `URI` in this class should be getting moved as part of move execution.

Reply:
We could probably come up with some type-fu to extract out stuff-to-move based on field types... not sure if it's worth it.